code/src/store/features/api/appApi.ts

```diff
@@ -0,0 +1,10 @@
+import { createApi, fetchBaseQuery } from "@reduxjs/toolkit/query/react";
+
+export const appApi = createApi({
+  baseQuery: fetchBaseQuery({
+    baseUrl: "http://localhost:5000/",
```

Comment:
Maybe we need to use `env` to configure `url`-s ? Now we have a dev deployment, does it work there?

Reply:
Yes, I think so. So I created an additional issue with setting this file